server/src/internalClusterTest/java/org/elasticsearch/gateway/ReplicaShardAllocatorSyncIdIT.java

```diff
@@ -243,6 +245,53 @@ public void testFullClusterRestartPerformNoopRecovery() throws Exception {
         assertNoOpRecoveries(indexName);
     }
 
+    /**
+     * If the recovery source is on an old node (before 7.2), then the recovery target won't have the safe commit after phase1
```

Comment:
Can we drop this test in v9? If so, can we link `Version#V_7_2_0` in the comment to remind us?

Reply:
++. I pushed https://github.com/elastic/elasticsearch/pull/57754/commits/b8d63fb4a00ab5958df4adbd710994d3fc92bc27.